app/models/project.rb

```diff
@@ -229,7 +229,7 @@ def async_sync(force_sync_dependencies: false)
     return unless platform_class_exists?
 
     sync_classes.each { |sync_class| PackageManagerDownloadWorker.perform_async(sync_class.name, name, nil, "project", 0, force_sync_dependencies) }
-    CheckStatusWorker.perform_async(id, status == "Removed" || status == "Deprecated")
+    CheckStatusWorker.perform_async(id)
```

Comment:
Removing the flag here should be appropriate, since we were just calculating the flag conditionally, and now we don't need to do that anymore.